src/views/Verbose.vue

```diff
@@ -40,7 +40,7 @@
                 {{ $t('verbose.count') }}
               </td>
               <td>
-                {{ verboseData.metadata.count.matched }} / {{ verboseData.metadata.count.total }}
+                {{ filteredNodeCount() }} / {{ verboseData.metadata.count.total }}
```

Comment:
Computed properties shouldn't have parenthesis, otherwise you get this error:

<img width="357" alt="image" src="https://user-images.githubusercontent.com/10846855/193377662-ad3bfc09-43d1-4a1a-86e5-20dbedc48aaa.png">

Reply:
Thank you. ❤️ 😻